ext/sqlite3/sqlite3.c

```diff
@@ -79,6 +79,7 @@ static void php_sqlite3_error(php_sqlite3_db_object *db_obj, char *format, ...)
 */
 PHP_INI_BEGIN()
 	STD_PHP_INI_ENTRY("sqlite3.extension_dir",  NULL, PHP_INI_SYSTEM, OnUpdateString, extension_dir, zend_sqlite3_globals, sqlite3_globals)
+	STD_PHP_INI_ENTRY("sqlite3.defensive",  "1", PHP_INI_SYSTEM, OnUpdateBool, dbconfig_defensive, zend_sqlite3_globals, sqlite3_globals)
```

Comment:
IMHO better to make this conditional, as if will be strange to expose a configuration option (in reflecton) which will be ignored at runtime.

Reply:
Good point, should be solved in 1bf542c659 thanks!